.zuul.yaml

```diff
@@ -3,15 +3,19 @@
     name: stf-base
     # defined in: https://review.rdoproject.org/cgit/config/tree/zuul.d/_jobs-crc.yaml
     parent: base-simple-crc
+    # need to set cluster-monitoring
     abstract: true
```

Comment:
No longer true due to latest changes landed on master.

Reply:
Yes. This PR is going to be wittled down to basically nothing as the other related changes get merged. This one is sticking around to test multiple changes together, so I'm not force pushing to other branches and making it difficult to review